src/hooks/useCioAutocomplete.ts

```diff
@@ -145,6 +163,28 @@ const useCioAutocomplete = (options: UseCioAutocompleteOptions) => {
       className: 'cio-form',
       'data-testid': 'cio-form',
     }),
+    getSectionProps: (section: Section) => {
+      const sectionName = section?.displayName || section?.identifier;
+      let attributes: React.DetailedHTMLProps<React.HTMLAttributes<any>, any> = {
+        className: `${sectionName} cio-section`,
+        ref: section.ref,
+        role: 'none',
+      };
+
+      const cnstrcDataAttributes = {
+        'data-cnstrc-recommendations': true,
+        'data-cnstrc-section': section.data[0]?.section,
+        'data-cnstrc-recommendations-pod-id': section.identifier,
+      };
+
+      if (section.type === 'recommendations') {
+        attributes = {
+          ...attributes,
+          ...cnstrcDataAttributes,
+        };
+      }
```

Comment:
@mocca102, can we move this key value pair to the object initially assigned to `attributes` above, or is only relevant to recommendations sections?
`'data-cnstrc-section': section.data[0]?.section,`

Would also refactor to include rec specific attributes inside of the condition and logical branching that explains these differences. How do we feel about something like this:

```javascript
      if (section.type === 'recommendations') {
         attributes['data-cnstrc-recommendations'] = true;
         attributes['data-cnstrc-recommendations-pod-id'] = section.identifier;
      }
```

Reply:
Done https://github.com/Constructor-io/constructorio-ui-autocomplete/pull/107/commits/55c168e54f3bc1568827cba615592e193f2fc3c1#diff-7f1c987b478566cc9cf82a2afc2ee7a7a150ec6047a7801e0ac01c410f7ce453R183